boa/test/strategies.py

```diff
@@ -62,7 +64,7 @@ def wrapper(*args: Tuple, exclude: Any = None, **kwargs: int) -> SearchStrategy:
 
 def _check_numeric_bounds(
     type_str: str, min_value: NumberType, max_value: NumberType
-) -> Tuple:
+) -> tuple[int | float, int | float]:
```

Comment:
i don't think this type signature is even right. it should return some kind of hypothesis strategy.

Reply:
oops, missed some hidden lines. this seems fine, although i think we can get more specific. either `tuple[NumberType, NumberType]` or `tuple[int, int]`